### PR TITLE
Json config excel

### PR DESCRIPTION
**Pull request by bhalonen**
Gutted old approach and moved to json based approach. Little easier to understand. 

# Related PR's
- [SurePulse](https://github.com/relinc/SurePulse/pull/47)

### CODE REVIEW

**Comment by markhalonen** [comment]
For EPPLus, there is only one `dll`, stored under `Libraries/`. This PR adds a ton of `dll` and other files under the `package/` directory for the json library. Any reason for the disparity?

**Comment by bhalonen** [comment]
I think I can cut everything but what is referenced here:
```
    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL"> 
      <HintPath>..\packages\Newtonsoft.Json.12.0.1\lib\net45\Newtonsoft.Json.dll</HintPath> 
    </Reference> 
```

File: ExcelMakerConsole/ExcelMakerConsole.csproj [hunk 1]
```diff
@@ -55,15 +55,24 @@
...
   </PropertyGroup>
   <PropertyGroup>
     <SignManifests>true</SignManifests>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetZone>LocalIntranet</TargetZone>
+  </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationManifest>Properties\app.manifest</ApplicationManifest>
+  </PropertyGroup>
```
Comment (markhalonen) [inline ExcelMakerConsole/ExcelMakerConsole.csproj:68]: I think these changes are giving me this error

image

pry just junk from the all-powerful IDE

we can pry just commit the app.manifest

File: ExcelMakerConsole/Sample.cs [hunk 2]
```diff
@@ -3,24 +3,38 @@
...
-        public DataColumn column1;
-        public DataColumn column2;
-        public DataColumn column3;
-        public DataColumn column4;
+        public DataColumn strain;
+        public DataColumn strainRate;
+        public DataColumn stress;
+        public DataColumn time;
+        public DataColumn frontFaceForce = null;
+        public DataColumn backFaceForce = null;
```
Comment (markhalonen) [inline ExcelMakerConsole/Sample.cs:18]: :+1:

**Comment by markhalonen** [comment]
:+1: looking pretty solid, definitely a big upgrade. 

- Indentation is not consistent
- Face force isn't exported to summary excel sheet

Other than that, looks good to me.

**Pull request merged**